akka-edge-docs/src/main/paradox/feature-summary.md

```diff
@@ -48,10 +48,13 @@ This case describes when the edge service is the consumer of events.
 * New events are emitted from the producer.
 * The stream remains open until either side chooses to close it, or there is an error.
 
-The offsets on the consumer side are stored in a database, but if the environment for the edge service doesn't
-have the capabilities to run a full database it can be a single Akka node with embedded @extref[H2 database](akka-persistence-r2dbc:getting-started.html#using-h2).
-The file mode of H2 is recommended to reduce memory requirements. That means that if the edge service is restarted
+The offsets on the consumer side are stored, but if the environment for the edge service does not
+have the capabilities to run a full database it can be a single Akka node with an embeddable storage such as the @extref[H2 database](akka-persistence-r2dbc:getting-started.html#using-h2).
+
+@@@ note
+When using H2, the file mode is recommended to reduce memory requirements. That means that if the edge service is restarted
 with a different file system all previous events will be delivered to the consumer again.
+@@@
```

Comment:
Not great with two boxes after each other like this (warning breakout next). Maybe make it just a regular but separate paragraph?

Reply:
Resolved in commit 87567577cc1a5f0433f7de7ae1272c0c21995cc5